requests/compat.py

```diff
@@ -33,6 +33,7 @@
 except ImportError:
     import json
 
+
```

Comment:
These spacing changes aren't really related to the current PR, can we remove them.

Reply:
I'll remove that.

Reply:
@nateprewitt I made the two above changes in my most recent commit